examples/demo-rollup/tests/bank/mod.rs

```diff
@@ -67,14 +68,22 @@ async fn send_test_create_token_tx(rpc_address: SocketAddr) -> Result<(), anyhow
 async fn bank_tx_tests() -> Result<(), anyhow::Error> {
     let (port_tx, port_rx) = tokio::sync::oneshot::channel();
 
+    // Use a dummy `elf` file, since the prover doesn't currently use it in native execution
+    let prover = Risc0Host::new(&[]);
+
     let rollup_task = tokio::spawn(async {
-        start_rollup(port_tx).await;
+        start_rollup(port_tx, Some(prover)).await;
     });
 
     // Wait for rollup task to start:
     let port = port_rx.await.unwrap();
 
-    send_test_create_token_tx(port).await?;
-    rollup_task.abort();
+    // If the rollup throws an error, stop trying to send the transaction and return it
+    tokio::select! {
+        err = rollup_task =>  {
+            err?;
+        },
+        res = send_test_create_token_tx(port) =>{res?},
```

Comment:
```suggestion
        res = send_test_create_token_tx(port) => {
            res?;
        },
```
just for consistency with an arm above